packages/react/src/components/Menu/MenuDivider.tsx

```diff
@@ -70,6 +70,6 @@ class MenuDivider extends UIComponent<WithAsProp<MenuDividerProps>> {
 MenuDivider.create = createShorthandFactory({ Component: MenuDivider, mappedProp: 'content' })
 
 /**
- * A menu divider visually segments menu items inside menu.
+ * A MenuDivider is non-focusable element that visually segments items of Menu.
```

Comment:
Can we replace the `non-focusable` with `non-actionable` element, as in that case it will be easy to reason what is the mean difference between `MenuItem` and `MenuDivider`

Reply:
have replace all the occurrences of `non-focusable` to `non-actionable`